mmcv/fileio/file_client.py

```diff
@@ -473,33 +473,47 @@ def __init__(self,
                  lock=False,
                  readahead=False,
                  **kwargs):
-        try:
-            import lmdb
-        except ImportError:
-            raise ImportError('Please install lmdb to enable LmdbBackend.')
-
         self.db_path = str(db_path)
-        self._client = lmdb.open(
-            self.db_path,
-            readonly=readonly,
-            lock=lock,
-            readahead=readahead,
-            **kwargs)
+        self.readonly = readonly
+        self.lock = lock
+        self.readahead = readahead
+        self.kwargs = kwargs
 
     def get(self, filepath):
         """Get values according to the filepath.
 
         Args:
             filepath (str | obj:`Path`): Here, filepath is the lmdb key.
         """
-        filepath = str(filepath)
-        with self._client.begin(write=False) as txn:
-            value_buf = txn.get(filepath.encode('ascii'))
-        return value_buf
+        return self[filepath]
 
     def get_text(self, filepath, encoding=None):
         raise NotImplementedError
 
+    def __getitem__(self, index):
+        if not hasattr(self, 'env'):
+            self.env = self._get_env()
+
+        with self.env.begin(write=False) as txn:
+            value_buf = txn.get(str(index).encode('utf-8'))
+        return value_buf
+
+    def _get_env(self):
+        try:
+            import lmdb
+        except ImportError:
+            raise ImportError('Please install lmdb to enable LmdbBackend.')
```

Comment:
Can we move this `try import` to `__init__` and just `import lmdb` in `_get_env`?